src/modules/rating/services/rating-scheduler.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable, Logger, OnModuleInit } from '@nestjs/common'
+import { CronJob } from 'cron'
+import { SchedulerRegistry } from '@nestjs/schedule'
+import { ConfigService } from '@nestjs/config'
+import { RatingRepository } from '../database/rating.repository'
+import { RatingConfig } from 'src/config/rating.config'
+
+@Injectable()
+export class RatingScheduler implements OnModuleInit {
+	private readonly logger = new Logger(RatingScheduler.name)
+	private readonly cronSchedule: string
+
+	constructor(
+		private readonly ratingRepository: RatingRepository,
+		private readonly schedulerRegistry: SchedulerRegistry,
+		private readonly configService: ConfigService<RatingConfig>,
```

Comment:
You still not follow **Step 3** in the guideline
https://github.com/patecan-io-software/trn-ecom-phucuong-inventory/pull/31#issuecomment-2049192754